src/digitalIdentity-frontend/src/app/pages/credential/credential-overview-page/credDef-overview-page.component.html

```diff
@@ -1,6 +1,6 @@
 <h1>Overview of credential definitions</h1>
 
-<button mat-raised-button routerLink="/create-credDef">
+<button mat-raised-button routerLink="/create-credential">
```

Comment:
create-credential-definition
(or "create-credDef", but use it everywhere the same way)